src/vm/ast/errors.rs

```diff
@@ -46,7 +46,7 @@ pub enum ParseErrors {
 pub struct ParseError {
     pub err: ParseErrors,
     pub pre_expressions: Option<Vec<PreSymbolicExpression>>,
-    pub diagnostic: Diagnostic,
+    pub diagnostic:Diagnostic,
```

Comment:
This would have been an issue in clarity :trollface:

Reply:
That'd be nice! I wouldn't have to make another commit to fix this monstrosity 😃